flexbox/src/main/java/com/google/android/flexbox/FlexboxLayoutManager.java

```diff
@@ -189,8 +206,9 @@ public FlexboxLayoutManager(@FlexDirection int flexDirection) {
      * @param flexDirection the flex direction attribute
      * @param flexWrap      the flex wrap attribute
      */
-    public FlexboxLayoutManager(@FlexDirection int flexDirection,
+    public FlexboxLayoutManager(Context context, @FlexDirection int flexDirection,
             @FlexWrap int flexWrap) {
+        setDisplayMetrics(context);
```

Comment:
Deleted the mDisplayWidth/mDisplayLength memebers. Now FlexboxLayoutManager has a context as its member and retrieves the display length when needed.